internal/resolver/resolver_test.go

```diff
@@ -85,6 +85,29 @@ func TestResolve(t *testing.T) {
 			},
 			err: true,
 		},
+		{
+			name: "repo from valid path under charts path",
+			req: []*chart.Dependency{
+				{Name: "localdependency", Repository: "", Version: "0.1.0"},
+			},
+			expect: &chart.Lock{
+				Dependencies: []*chart.Dependency{
+					{Name: "localdependency", Repository: "", Version: "0.1.0"},
+				},
+			},
+		},
+		{
+			name: "repo from valid path under charts path",
```

Comment:
I think this test name needs to be "repo from _invalid_ path under charts path" correct?

Reply:
Sure. Done.